Foundation/GTMNSDictionary+URLArguments.h

```diff
@@ -29,12 +29,12 @@
 /// NOTE: Apps targeting iOS 8 or OS X 10.10 and later should use
 ///       NSURLComponents and NSURLQueryItem to create URLs with
 ///       query arguments instead of using these category methods.
-+ (NSDictionary *)gtm_dictionaryWithHttpArgumentsString:(NSString *)argString;
++ (NSDictionary *)gtm_dictionaryWithHttpArgumentsString:(NSString *)argString NS_DEPRECATED(10_0, 10_10, 2_0, 8_0, "Use NSURLComponents and NSURLQueryItem.");
```

Comment:
This one I did wonder if we should do a second impl on top of the newer apis directly, but I guess it isn't worth it.